src/cdk/drag-drop/drop-list-ref.ts

```diff
@@ -335,9 +336,11 @@ export class DropListRef<T = any> {
    *    container when the item was dropped.
    * @param distance Distance the user has dragged since the start of the dragging sequence.
    * @breaking-change 9.0.0 `distance` parameter to become required.
+   *    10.0.0 `event` parameter to become required.
```

Comment:
Note that this looks a little weird, but dgeni was throwing if I add it into a separate `@breaking-change`.